src/lib/message/L1ToL2Message.ts

```diff
@@ -461,17 +485,28 @@ export class L1ToL2MessageReader extends L1ToL2Message {
    */
   public async waitForStatus(
     confirmations?: number,
-    timeout = 900000
+    timeout?: number
   ): Promise<L1ToL2MessageWaitResult> {
+    const chosenTimeout = isDefined(timeout)
+      ? timeout
+      : this.getDepositTimeout(this.chainId)
+
     // try to wait for the retryable ticket to be created
     const _retryableCreationReceipt = await this.getRetryableCreationReceipt(
       confirmations,
-      timeout
+      chosenTimeout
     )
-    if (!_retryableCreationReceipt)
-      throw new ArbSdkError(
-        `Retryable creation receipt not found ${this.retryableCreationId}`
-      )
+    if (!_retryableCreationReceipt) {
+      if (confirmations || chosenTimeout) {
+        throw new ArbSdkError(
+          `Timed out waiting to retrieve retryable creation receipt: ${this.retryableCreationId}.`
+        )
+      } else {
+        throw new ArbSdkError(
+          `Retryable creation receipt not found ${this.retryableCreationId}.`
+        )
+      }
+    }
```

Comment:
these unrelated change are still here

Reply:
I did reset to c-nitro-stable and force push 
Not sure why they're still there tho